### PR TITLE
Concept exercise review: poetry-club-door-policy

### PR DESCRIPTION
**Pull request by angelikatyborska**
Hello! I solved this exercise for the first time just now and I would like to propose some tiny fixes.

I also opened an issue for a problem that I didn't feel comfortable fixing myself (https://github.com/exercism/javascript/issues/1224)

### CODE REVIEW

File: exercises/concept/poetry-club-door-policy/.docs/instructions.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Because there have been incidents in the past, the club has a very specific door
 There are two doors at the poetry club, both are guarded.
 In order to gain entry, you'll need to work out the password of that day:
 
-## Front door
+**Front door**
```
Comment (angelikatyborska) [inline exercises/concept/poetry-club-door-policy/.docs/instructions.md:9]: It sucks but this can't be a heading... any h2 will be interpreted as a task:

Screen Shot 2021-07-24 at 18 26 35

Comment (SleeplessByte) [reply exercises/concept/poetry-club-door-policy/.docs/instructions.md:9]: Ah yes. See, it's THAT old ;)

File: exercises/concept/poetry-club-door-policy/.docs/instructions.md [hunk 2]
```diff
@@ -96,6 +96,6 @@ backDoorResponse('Stands so high   ');
 Change the implementation of the `backDoorPassword` function so that it's polite:
 
 ```javascript
-backDoorPassword('esroh');
+backDoorPassword('horse');
```
Comment (SleeplessByte) [inline exercises/concept/poetry-club-door-policy/.docs/instructions.md:99]: I think `esroh` was right.

Comment (angelikatyborska) [reply exercises/concept/poetry-club-door-policy/.docs/instructions.md:99]: if `esroh` was right then the exemplar and the test suite are wrong because neither of those expect the string to be reversed: https://github.com/exercism/javascript/blob/main/exercises/concept/poetry-club-door-policy/.meta/exemplar.js#L61-L63

Comment (SleeplessByte) [reply exercises/concept/poetry-club-door-policy/.docs/instructions.md:99]: Ha yes. Of course. Stupid.

It _does_ reverse it, but it does that as part of the test suite. aka, the _guards_ give you the sentences reversed, and `backDoorResponse` picks the _reversed_ first letter, which are then passed into `backdoorPassword`. So by the time they get to this, it's no longer reversed.

TL;DR: you're right.

**Review by SleeplessByte — APPROVED** [review]
